eo-maven-plugin/src/main/java/org/eolang/maven/rust/PrimeModule.java

```diff
@@ -58,4 +58,32 @@ public PrimeModule(final String method, final String file) {
             file
         );
     }
+
+    /**
+     * Translates ("mangles") Java method name to native method name.
```

Comment:
@levBagryansky let's add an example to the doc. Something like "before -> after"